src/openapi/streaming/subscription.ts

```diff
@@ -45,24 +45,28 @@ export interface StreamingOptions {
      * @param data - data received
      * @param updateType - either be subscription.UPDATE_TYPE_DELTA or subscription.UPDATE_TYPE_SNAPSHOT
      */
-    onUpdate?: (data: unknown, updateType: SubscriptionUpdateTypes) => void;
+    onUpdate?: (
+        data: unknown,
+        updateType: SubscriptionUpdateTypes,
+        subscription: Subscription,

```

Comment:
please update JSDoc for every function that now accepts subscription as a parameter

Reply:
Fixed